scripts/storage-emulator-integration/tests.ts

```diff
@@ -225,10 +225,14 @@ describe("Storage emulator", () => {
 
           expect(fileMetadata).to.deep.include(metadata);
         });
-
+        // TODO(b/241151246): Fix conformance tests.
         it("should handle resumable upload with name only in metadata", async () => {
+          const expectedHost = TEST_CONFIG.useProductionServers
```

Comment:
You can probably extract this to the top level `describe` to save yourself from having to declare this in every test case